CHANGELOG.md

```diff
@@ -329,222 +399,283 @@ Bugfixes:
-* introduce Y031 (use class-based syntax for `TypedDict`s where possible)
-* all errors are now enabled by default
-* remove Y092 (top-level attribute must not have a default value)
-* `attrs` is no longer a dependency
-* `ast_decompiler` has been added as a dependency on Python 3.8 and 3.7
-* support Python 3.10
-* discontinue support for Python 3.6
+* Detect usage of non-integer indices in `sys.version_info` checks.
+
+### Other Changes
+
+* All errors are now enabled by default.
+* `attrs` is no longer a dependency.
+* `ast_decompiler` has been added as a dependency on Python 3.8 and 3.7.
+* Support Python 3.10.
+* Discontinue support for Python 3.6.
+
+## [20.10.0]
+
+### Other Changes
+
+* Support Python 3.9.
+
+## [20.5.0]
+
+### New Error Codes
+
+* Y015 – Attribute must not have a default value other than `...`
+* Y091 – Function body must not contain `raise`
+* Y092 – Top-level attribute must not have a default value
+
+### Other Changes
+
+* Support flake8 3.8.0.
+
+## [19.3.0]
+
+### Other Changes
+
+* Update pyflakes dependency.
 
-## 20.10.0
+## [19.2.0]
 
-* support Python 3.9
+### New Error Codes
 
-## 20.5.0
+* Y012 – Class body must not contain `pass`
+* Y013 – Non-empty class body must not contain `...`
+* Y014 – Only simple default values are allowed for any function arguments
 
-* support flake8 3.8.0
-* introduce Y091 (function body must not contain `raise`)
-* introduce Y015 (attribute must not have a default value other than `...`)
-* introduce Y092 (top-level attribute must not have a default value)
+### Other Changes
 
-## 19.3.0
+* Support Python 3.7.
+* Use `--stdin-display-name` as the filename when reading from stdin.
 
-* update pyflakes dependency
+## [18.3.1]
 
-## 19.2.0
+### New Error Codes
 
-* support Python 3.7
-* add a check for non-ellipsis, non-typed arguments
-* add checks for checking empty classes
-* use --stdin-display-name as the filename when reading from stdin
+* Y011 – Allow only simple default values
 
-## 18.3.1
+## [18.3.0]
 
-* introduce Y011
+Release herp derp, don't use.
 
-## 18.3.0
+## [17.3.0]
 
-* (release herp derp, don't use)
+### New Error Codes
 
-## 17.3.0
+* Y001 – Y010
+* Y090 (disabled by default)
 
-* introduce Y001 - Y010
-* introduce optional Y090
+## [17.1.0]
 
-## 17.1.0
+### Other Changes
 
-* handle `del` statements in stub files
+* Handle `del` statements in stub files.
 
-## 16.12.2
+## [16.12.2]
 
-* handle annotated assignments in 3.6+ with forward reference support
+### Other Changes
 
-## 16.12.1
+* Handle annotated assignments in 3.6+ with forward reference support.
 
-* handle forward references during subclassing on module level
+## [16.12.1]
 
-* handle forward references during type aliasing assignments on module level
+### Other Changes
 
-## 16.12.0
+* Handle forward references during subclassing on module level.
+* Handle forward references during type aliasing assignments on module level.
```

Comment:
I think these things also relate to the pyflakes monkeypatching that we no longer do; it might be worth explicitly stating that